packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+---
+title: SCIM 1.1 Protocol Reference
+icon: /assets/img/icons/scim.svg
+meta:
+  - name: description
+    content: Your SCIM API must support specific SCIM 1.1 API endpoints to work with Okta. Those endpoints and their explanations are detailed here.
+---
+
+# Okta and SCIM 1.1 Protocol
+
+This reference focuses on how Okta API endpoints share information with SCIM Version 1.1 specific API calls.
+
+The SCIM Protocol is an application-level, REST protocol for provisioning and managing identity data on the web. The protocol supports creation, discovery, retrieval, and modification of core identity Resources.
+
+## User operations
+
+### Creating users
```

Comment:
From the headings, at least, it's not really clear to me how this differs from `Create the user` below.

Reply:
I cleaned up the create user section to hopefully make it flow better.